src/Sylius/Bundle/ResourceBundle/Doctrine/ORM/EntityRepository.php

```diff
@@ -101,6 +102,28 @@ public function findBy(array $criteria, array $sorting = array(), $limit = null,
     /**
      * {@inheritdoc}
      */
+    public function add(ResourceInterface $resource)
+    {
+        $this->_em->persist($resource);
+
```

Comment:
I think we should remove them all, sorry for spam, I thought it is just 2 places. :)